js/anims-lasers.js

```diff
@@ -0,0 +1,30 @@
+$(function() {
+    $('.city').click(function () {
+        shootLaser($(this))
+    })
+})
+
+document.addEventListener('keypress', function(e) {
+    switch (true) {
```

Comment:
look at the code in patrick and buraks branch

Reply:
fixed in 128e995